src/lib/logger.js

```diff
@@ -2,7 +2,26 @@ import util from 'util';
 
 import { store } from '../store';
 
-const { localStorage } = window;
+const isMobile = () => {
+	let hasTouchScreen = false;
+	if ('maxTouchPoints' in navigator) {
+		hasTouchScreen = navigator.maxTouchPoints > 0;
+	} else if ('msMaxTouchPoints' in navigator) {
+		hasTouchScreen = navigator.msMaxTouchPoints > 0;
+	} else {
+		const mQ = window.matchMedia && matchMedia('(pointer:coarse)');
+		if (mQ && mQ.media === '(pointer:coarse)') {
+			hasTouchScreen = !!mQ.matches;
+		} else if ('orientation' in window) {
+			hasTouchScreen = true;
+		} else {
+			const UA = navigator.userAgent;
+			hasTouchScreen = /\b(BlackBerry|webOS|iPhone|IEMobile)\b/i.test(UA)
+              || /\b(Android|Windows Phone|iPad|iPod)\b/i.test(UA);
+		}
+	}
+	return hasTouchScreen;
+};
```

Comment:
Thanks @bhardwajaditya this worked when I tested it on my phone....  I see a lot of conditions here......  I assume it should work to detect all mobile devices?

Reply:
Yes it will work, this is based on mdn specifications